### DISCUSSION TITLE
Improve error message when miner snapshot load can't find file

### DISCUSSION THREAD

**Discussion by rootulp** [helium/miner#1663] [Ideas]
Currently the error message for `miner snapshot load` doesn't clearly state that the snapshot file couldn't be found. Is it possible to improve this error message?

```
miner snapshot load validator_data/snap-1283041
RPC to 'val_miner@127.0.0.1' failed: {'EXIT',
                                      {{{badmatch,{error,enoent}},
                                        [{blockchain_ledger_snapshot_v1,
                                          deserialize,2,
                                          [{file,
                                            "blockchain_ledger_snapshot_v1.erl"},
                                           {line,440}]},
                                         {blockchain_worker,handle_call,3,
                                          [{file,"blockchain_worker.erl"},
                                           {line,428}]},
                                         {gen_server,try_handle_call,4,
                                          [{file,"gen_server.erl"},
                                           {line,721}]},
                                         {gen_server,handle_msg,6,
                                          [{file,"gen_server.erl"},
                                           {line,750}]},
                                         {proc_lib,wake_up,3,
                                          [{file,"proc_lib.erl"},
                                           {line,236}]}]},
                                       {gen_server,call,
                                        [blockchain_worker,
                                         {install_snapshot_from_file,
                                          "validator_data/snap-1283041"},
                                         infinity]}}}
```

**Comment by serbyxp** [comment]
Did you sync pause before you did snapshot load? Also ARM64 has a lot of bugs / lags in the miner console that “timeout” and what not, VS AMD64 which usually handles the Json RPC a lot better, I’m not 100% sure but if you don’t have your port exposed on container , for RPC and are trying to do send commands from outside of the container, it can cause issues. Also whatever your using to handle your containers , IE docker , needs to have the right privilege mode etc, if your trying to run it out of the container on the “local host” . I have it setup like that so I don’t know how it would react outside of those parameters… but that’s all in whatever container manager documentation your using. Takes some fiddling around with. But yeah descriptive error messages would be helpful, instead of having to go through the repo and finding all those lines to see what the error is…

**Comment by rootulp** [comment]
Thanks for the tips @serbyxp ! In this instance, I think `miner snapshot load` expects an absolute path to the snapshot file but when you give it a relative path, it fails with an unhelpful error message.
